shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -566,7 +550,7 @@ export default {
       />
     </div>
 
-    <div v-if="!hasV1Monitoring && componentServices">
+    <div v-if="clusterServices">
```

Comment:
should be `componentServices`

Reply:
Looks like this was the last difference I could spot. Really good eye man. Thanks.